src/client.ts

```diff
@@ -19,17 +22,38 @@ interface ClientConfigOptions {
   clientWindow?: Window;
 }
 
+/**
+ * The known events that can be emitted
```

Comment:
This is a little annoying because this is another definition of the ClientToHost messages we have.